Lib/test/test_audit.py

```diff
@@ -19,7 +19,7 @@ class AuditTest(unittest.TestCase):
     maxDiff = None
 
     @support.requires_subprocess()
-    def run_test_in_subprocess(self, *args):
```

Comment:
Is it related to regrtest?

Reply:
nope! Not sure how that slipped in, must have committed it by mistake :)

It's gone now!